File: MP1/StudentInfo.py
list = []
class Student:
    def __init__(self, studentName:str, studentAge:int, studentGPA:float, studentGradeLevel:str):
        self.__name = studentName
        self.__age = studentAge
        self.__gpa = studentGPA
        self.__gradelevel = studentGradeLevel 
    
    def enroll(self, name, age, gpa, gradeLevel):
        enrollee = Student(name, age, gpa, gradeLevel)
        list.append(enrollee)
    #Testing purposes
    #def display(self, enrollee):
    #    print("Name        : ", enrollee.__name)
    #    print("Age         : ", enrollee.__age)
    #    print("GPA         : ", enrollee.__gpa)
    #    print("Grade Level : ", enrollee.__gradelevel)
    #    print("\n") 

#s:Student = Student('', 0,0, '')
#s.enroll("Mark", 18, 1.5, "Freshie")
#s.enroll("Angel", 19, 1.0, "Sophomore")
#s.enroll("Charles", 20, 1.45, "Junior")

#for i in range(list.__len__()):
#    s.display(list[i])

File: MP1/StudentInfo.cpp
#include <iostream>
#include <iomanip> 
using namespace std;


class Student{
    private:
        char name[50];
        int age;
        float gpa;
        char grade[10];
    public:
        void enroll(){
            cout << "Enter name: ";
            cin >> name;
            cout << "Enter age: ";
            cin >> age;
            cout << "Enter gpa: ";
            cin >> gpa;
            cout << "Enter grade: ";
            cin >> grade;
        }
        /*void display(){
            cout << "Name: " << name << endl;
            cout << "Age: " << age << endl;
            cout << "GPA: " << setprecision(2) << gpa << endl;
            cout << "Grade: " << grade << endl;
        }*/
};

int main(){
    int n;
    Student s[50];

    cout << "Enter number of students: ";
    cin >> n;

    for(int i = 0; i < n; i++){
        s[i].enroll();
    }
    /* 
    for(int i = 0; i < n; i++){
        s[i].display();
    }
    */
    return 0;
}